bot/cogs/spookyavatar.py

```diff
@@ -0,0 +1,52 @@
+from io import BytesIO
+import os
+
+import aiohttp
+import discord
+from discord.ext import commands
+from PIL import Image
+
+from bot.resources import spookifications
+
+
+class SpookyAvatar:
+
+    """
+    A cog that spookifies an avatar.
+    """
+
+    def __init__(self, bot):
+        self.bot = bot
+
+    async def get(self, url):
+        """
+        Returns the contents of the supplied url.
+        """
+        async with aiohttp.ClientSession() as session:
+            async with session.get(url) as resp:
+                return await resp.read()
+
+    @commands.command(name='savatar', aliases=['spookyavatar', 'spookify'],
+                      brief='Spookify an user\'s avatar.')
+    async def spookyavatar(self, ctx, user: discord.Member=None):
```

Comment:
Per Flake8:

```python
async def spookyavatar(self, ctx, user: discord.Member = None):
```

For the third time, please run Flake8 before committing towards this repository

Reply:
Flake8 doesn't report an error here. 
Anyway, for default parameters values, whitespaces shouldn't be used acc to (PEP8)[https://legacy.python.org/dev/peps/pep-0008/#other-recommendations]

Reply:
Travis disagrees:

```
6.46s$ pip install flake8 pipenv salt-pepper
Collecting flake8
  Downloading https://files.pythonhosted.org/packages/34/a6/49e2849a0e5464e1b5d621f63bc8453066f0f367bb3b744a33fca0bc1ddd/flake8-3.6.0-py2.py3-none-any.whl (68kB)
    100% |████████████████████████████████| 71kB 2.5MB/s 
Collecting pipenv
  Downloading https://files.pythonhosted.org/packages/90/06/0008f53835495fbbf6e31ced9119b8f517e1271bdefcf0d04aaa9f28dbf4/pipenv-2018.10.13-py3-none-any.whl (5.2MB)
    100% |████████████████████████████████| 5.2MB 303kB/s 
Collecting salt-pepper
  Downloading https://files.pythonhosted.org/packages/1e/b6/fc53b0d5a83f913668f03b98f1cb642774bc27ba0c08c306b92d9a549196/salt-pepper-v0.7.2.tar.gz
Collecting pyflakes<2.1.0,>=2.0.0 (from flake8)
  Downloading https://files.pythonhosted.org/packages/44/98/af7a72c9a543b1487d92813c648cb9b9adfbc96faef5455d60f4439aa99b/pyflakes-2.0.0-py2.py3-none-any.whl (53kB)
    100% |████████████████████████████████| 61kB 9.8MB/s 
Collecting pycodestyle<2.5.0,>=2.4.0 (from flake8)
  Downloading https://files.pythonhosted.org/packages/e5/c6/ce130213489969aa58610042dff1d908c25c731c9575af6935c2dfad03aa/pycodestyle-2.4.0-py2.py3-none-any.whl (62kB)
    100% |████████████████████████████████| 71kB 7.9MB/s 
Requirement already satisfied: setuptools>=30 in /home/travis/virtualenv/python3.7-dev/lib/python3.7/site-packages (from flake8)
Collecting mccabe<0.7.0,>=0.6.0 (from flake8)
  Downloading https://files.pythonhosted.org/packages/87/89/479dc97e18549e21354893e4ee4ef36db1d237534982482c3681ee6e7b57/mccabe-0.6.1-py2.py3-none-any.whl
Requirement already satisfied: pip>=9.0.1 in /home/travis/virtualenv/python3.7-dev/lib/python3.7/site-packages (from pipenv)
Collecting virtualenv-clone>=0.2.5 (from pipenv)
  Downloading https://files.pythonhosted.org/packages/16/9d/6419a4f0fe4350db7fdc01e9d22e949779b6f2d2650e4884aa8aededc5ae/virtualenv_clone-0.4.0-py2.py3-none-any.whl
Collecting certifi (from pipenv)
  Downloading https://files.pythonhosted.org/packages/56/9d/1d02dd80bc4cd955f98980f28c5ee2200e1209292d5f9e9cc8d030d18655/certifi-2018.10.15-py2.py3-none-any.whl (146kB)
    100% |████████████████████████████████| 153kB 5.5MB/s 
Collecting virtualenv (from pipenv)
  Downloading https://files.pythonhosted.org/packages/b6/30/96a02b2287098b23b875bc8c2f58071c35d2efe84f747b64d523721dc2b5/virtualenv-16.0.0-py2.py3-none-any.whl (1.9MB)
    100% |████████████████████████████████| 1.9MB 794kB/s 
Building wheels for collected packages: salt-pepper
  Running setup.py bdist_wheel for salt-pepper ... done
  Stored in directory: /home/travis/.cache/pip/wheels/c3/ab/10/72e41c7bf0a101b165134b91bb3074262dd8a904ea096cbc0f
Successfully built salt-pepper
Installing collected packages: pyflakes, pycodestyle, mccabe, flake8, virtualenv-clone, certifi, virtualenv, pipenv, salt-pepper
Successfully installed certifi-2018.10.15 flake8-3.6.0 mccabe-0.6.1 pipenv-2018.10.13 pycodestyle-2.4.0 pyflakes-2.0.0 salt-pepper-0.7.2 virtualenv-16.0.0 virtualenv-clone-0.4.0
install.2
13.72s$ pipenv install --deploy
Courtesy Notice: Pipenv found itself running within a virtual environment, so it will automatically use that environment, instead of creating its own for any project. You can set PIPENV_IGNORE_VIRTUALENVS=1 to force pipenv to ignore that environment and create its own instead. You can set PIPENV_VERBOSITY=-1 to suppress this warning.
Installing dependencies from Pipfile.lock (1b4c78)…
  🐍   ▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉ 3/3 — 00:00:13
1.12s$ pipenv run lint
Courtesy Notice: Pipenv found itself running within a virtual environment, so it will automatically use that environment, instead of creating its own for any project. You can set PIPENV_IGNORE_VIRTUALENVS=1 to force pipenv to ignore that environment and create its own instead. You can set PIPENV_VERBOSITY=-1 to suppress this warning.
bot/__main__.py:2:1: F401 'os.environ' imported but unused
bot/__main__.py:8:83: E261 at least two spaces before inline comment
bot/__main__.py:8:83: E262 inline comment should start with '# '
bot/cogs/spookyavatar.py:31:59: E252 missing whitespace around parameter equals
bot/cogs/spookyavatar.py:31:60: E252 missing whitespace around parameter equals
```

Reply:
As do the recommendations for type annotated defaults:

![image](https://user-images.githubusercontent.com/5323929/47567606-01017b80-d8fd-11e8-9789-65d1977f7b79.png)

Reply:
Oh okay, you're right. Updating.